src/hermit.js

```diff
@@ -212,15 +210,15 @@ export class HermitExportPublicKey extends HermitReader {
       throw new Error("No BIP32 path in QR code.");
     }
     result.bip32Path = bip32Path;
-    delete result.bip32_path;
+    Reflect.deleteProperty(result, "bip32_path");
```

Comment:
wow, I've never heard of `Reflect` before! Neat.

Reply:
I know this is just a lint cleanup, but what exactly is going on here? Do you know why there's this rigamarole with two different `bip32Path` properties?

Reply:
> wow, I've never heard of `Reflect` before! Neat.
Same -- this was an IDE mediated swap.

I'm not sure about the rigamarole, except looking at `parseHermitQRCodeData` looks like it grabs the data and tries to export JSON. To me this means that hermit exports labeled JSON into the QR code and the labels don't match the shape of the library (or other keystores). This looks like where that issue is corrected.

Reply:
Oh yeah, you're right. I think i remember Destry patching this previously to fix a bug :-/